ibm-cloud-pak-for-data/azuredeploy.json

```diff
@@ -276,18 +276,21 @@
 		},
 		"rhelUsername": {
 			"type": "string",
+			"defaultValue": "na",
```

Comment:
use an emptyString for the defaultValues

Reply:
@bmoore-msft I get a template error in your pipeline when using an empty string. It seems like templates don't support an empty string. Please advise.

Reply:
What's the error?

Reply:
I got this when  I tried (even with minLength=0) - https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=27151&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=7bf03a3b-69d5-5660-244f-21e634d6903f

Reply:
It looks like you have minLength on the parameters in the nested templates... those would need to be removed...

Reply:
@bmoore-msft Thanks for catching that. commited the changes!